DoomModLoader2C/Forms/FormMod.cs

```diff
@@ -152,7 +162,11 @@ private void cmdPlay_Click(object sender, EventArgs e)
             pro.StartInfo.FileName = sourcePort.path;
             pro.Start();
 
-
+            pro.WaitForExit();
```

Comment:
No need to lock the UI thread until the sourceport's process stop. When the the spawned software close `OnSourceportClosed` gets called. Currently it's only used to apply a workaround for gzdoom quicksaves via custom key bind (see https://github.com/Premo36/DML2.X/issues/14), but it can be used to do anything that needs to be done once the sourceport exits.

Reply:
I originally added this due to the mod launcher remaining on top of the gzdoom window when I first started playing around with the code; I don't seem to see this behaviour now and will remove this line.